var/spack/repos/builtin/packages/py-zstandard/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyZstandard(PythonPackage):
+    """FIXME: Put a proper description of your package here."""
+
+    homepage = "https://github.com/indygreg/python-zstandard"
+    pypi = "zstandard/zstandard-0.21.0.tar.gz"
+
+    version("0.21.0", sha256="f08e3a10d01a247877e4cb61a82a319ea746c356a3786558bed2481e6c405546")
+
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-cffi@1.11:", type=("build", "run"))
```

Comment:
I think cffi is only required in PyPy and is optional in CPython. Might want to add a variant instead. But I haven't tested anything.

Reply:
Is there any prior art here? (I couldn't find any.)

Is the suggested strategy (for now) to use a somewhat standard variant `cffi` or so, instead of a future `when="^pypy"`?

Reply:
Spack doesn't support PyPy, and until #17475 is solved, it can't. So for now you can ignore it.

In the case of this package, you can either remove the dep, or add a `+cffi` variant to control it.

Reply:
See #17475, but I think we can support that now if we choose to